src/streaming/protection/drm/KeySystemClearKey.js

```diff
@@ -33,7 +33,7 @@ import KeyPair from '../vo/KeyPair';
 import ClearKeyKeySet from '../vo/ClearKeyKeySet';
 import CommonEncryption from '../CommonEncryption';
 
-const uuid = '1077efec-c0b2-4d02-ace3-3c1e52e2fb4b';
+const uuid = 'e2719d58-a985-b3c9-781a-b030af78d30e';
```

Comment:
Correct me if I'm wrong, but wouldn't this be a breaking change for people relying on the existing, non-DASH-IF uuid?

Reply:
Yes you're correct in this regard.
I believe this is to follow the IOP spec regarding ClearKey which states that the `1077...` id shall not be used to indicate ClearKey usage.

I'm not entirely sure what the correct approach is here in terms of what we want to see going forward. Allowing both is technically against the IOP spec but I agree that it may be the correct implementation for this PR.

Reply:
I would perhaps suggest accepting both but emitting a warning in the log about it being IOP-non-conforming content when the W3C Common PSSH is used.

Reply:
I think both should be acceptable as Sander suggests, possibly with a warning. Given the existing architecture, this probably requires a new KeySystem implementation so there are two for clearkey - one for DASH-IF, one for W3C. In time the W3C implementation could possibly be deprecated, but I think removal would be inappropriate at this stage.